### PR TITLE
Adds empty project for the custom-tabs-oauth demo

### PR DESCRIPTION
**Pull request by andreban**
The project is based on the Basic Android app wizard, but with
tests dependencies and directories removed.

### CODE REVIEW

File: demos/custom-tabs-oauth/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/build
```
Comment (PEConn) [inline demos/custom-tabs-oauth/.gitignore:1]: nit: newlines

File: demos/custom-tabs-oauth/build.gradle [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ */
+
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 30
+    buildToolsVersion "30.0.1"
+
+    defaultConfig {
+        applicationId "com.google.browser.examples.custom_tabs_oauth"
```
Comment (PEConn) [inline demos/custom-tabs-oauth/build.gradle:24]: Do you mind making this `com.google.androidbrowserhelper.demos.customtabsoauth`? I'm trying to get all the demos to have packages like that - though I've not got around to a full on refactor yet.

Also happy to standardize on a different package if you've got suggestions.

Could you similarly change the references in the Java code and AndroidManifest.xml?

File: demos/custom-tabs-oauth/build.gradle [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        release {
+            minifyEnabled false
+        }
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: "libs", include: ["*.jar"])
+    implementation 'androidx.appcompat:appcompat:1.2.0'
+    implementation 'androidx.constraintlayout:constraintlayout:2.0.1'
```
Comment (PEConn) [inline demos/custom-tabs-oauth/build.gradle:41]: We can ideally get rid of both of these.

**Review by PEConn — COMMENTED** [review]
Is there any way to view images in GitHub's review tool? Personally (this is just me procrastinating) I'd kinda like each of the demos to have a nice launcher image instead of the Android one - for Play Billing I went to [Pexels](https://www.pexels.com) and got something vaguely nice.

Also, do you think it would be worth having a demo template project?

File: demos/custom-tabs-oauth/build.gradle (hunk 2 above)
Comment (andreban) [reply demos/custom-tabs-oauth/build.gradle:24]: Ha - I did notice we had some variation and went with one we already have. Will change.

**Comment by andreban** [comment]
> Is there any way to view images in GitHub's review tool? Personally (this is just me procrastinating) I'd kinda like each of the demos to have a nice launcher image instead of the Android one - for Play Billing I went to [Pexels](https://www.pexels.com) and got something vaguely nice.
> 
> Also, do you think it would be worth having a demo template project?

Hum, I don't know if there's an easy way - I went with the default icon for now, I'm thinking about something wit a lock. Pexels is new to me, will check it out. I like the idea of a template project, but should it be commited here?

**Comment by PEConn** [comment]
Nah, demo project can be a different PR (but it makes sense to do around now since we'll have the stuff ready.

File: demos/custom-tabs-oauth/.gitignore (hunk 1 above)
Comment (andreban) [reply demos/custom-tabs-oauth/.gitignore]: done

File: demos/custom-tabs-oauth/build.gradle (hunk 2 above)
Comment (andreban) [reply demos/custom-tabs-oauth/build.gradle]: renamed the id (and package)

File: demos/custom-tabs-oauth/build.gradle (hunk 3 above)
Comment (andreban) [reply demos/custom-tabs-oauth/build.gradle]: done

**Comment by andreban** [comment]
@PEConn PTAL

**Review by PEConn — APPROVED** [review]

**Pull request merged**